.github/workflows/ffi-builds.yml

```diff
@@ -165,10 +165,11 @@ jobs:
 
       - name: Copy/Build licenses
         run: |
-          echo "# livekit" > TEMP_LICENSE.md
-          echo "```" >> TEMP_LICENSE.md
+          echo '# livekit' > TEMP_LICENSE.md
+          echo '```' >> TEMP_LICENSE.md
           cat LICENSE >> TEMP_LICENSE.md
-          echo "```" >> TEMP_LICENSE.md
+          echo '```' >> TEMP_LICENSE.md
+          touch livekit-ffi/WEBRTC_LICENSE.md
```

Comment:
Regarding my comment about the .gitignore excluding the LICENSE file, simply creating an empty LICENSE file doesn’t seem like a good solution.

Reply:
Fair enough. I chose this route since, as seen [here](https://github.com/livekit/rust-sdks/blob/65a1b02a2806547fa1f77a55bab1e24eef194880/livekit-ffi/build.rs#L41), the file is only generated when the target is an Android device. Maybe a better alternative would be running this job when an Android target matches.